### ISSUE TITLE
Cannot open file 'frontend_cryptoTools/miracl.lib' and 'frontend/libboost_date_time-vc141-mt-gd-x64-1_69.lib'

### ISSUE THREAD

**Issue by sandy9999** [osu-crypto/libOTe#34]
![image](https://user-images.githubusercontent.com/17477947/94465191-254aee80-01dd-11eb-9a77-90ba281b2c30.png)

Is the above intended behaviour? I am using Windows 10 and this is my project.jam configuration

````
import option ; 
 
using msvc : 14.1 : "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\bin\Hostx86\x64\cl.exe";
 
option.set keep-going : false ; 
 ````

#### Build Boost
1) `git clone --recursive https://github.com/osu-crypto/libOTe.git`
2) `cd libOTe/cryptoTools/thirdparty/win`
3) Download boost 1.69.0 .zip file from https://www.boost.org/users/history/version_1_69_0.html inside this directory and extract
4) `$PWD\bootstrap.bat`
5) `.\b2.exe  toolset=msvc-14.0 architecture=x86 address-model=64 --with-thread --with-filesystem --with-regex --with-date_time stage link=static variant=debug,release runtime-link=static threading=multi`

#### Enable Relic
1) `git clone https://github.com/ladnir/relic.git`
2) `cd relic`
3) `cmake . -DMULTI=OPENMP -DCMAKE_INSTALL_PREFIX:PATH=C:\libs  -DCMAKE_GENERATOR_PLATFORM=x64 -DRUNTIME=MT`
4) `cmake --build .`
5) `cmake --install .`
6) Edit the config file libOTe/cryptoTools/cryptoTools/Common/config.h to include #define ENABLE_RELIC.

#### Enable Miracl
1) cd libOTe/cryptoTools/thirdparty/win
2) `git clone https://github.com/ladnir/miracl.git`
3) Built Miracl.sln in Visual Studio 2019
4) Edit the config file libOTe/cryptoTools/cryptoTools/Common/config.h to include #define ENABLE_MIRACL

There were minor bugs and config changes along the way which I had to fix to make it compatible with Visual Studio 2019.

Please let me know if you need any other details to reproduce the error.

**Comment by ladnir** [comment]
You probably don't want to enable both relic and miracl. They serve the same purpose. I would suggest only enabling relic.

For boost, it seems you have not built the required library or its not in the correct location. It appears that you edited the project-config.jam file to use MSVC 14.1 while you command for b2.exe has MSVC 14.0. Thats likely causing the error.

libOTe is expecting the boost library to be located at one of the following

- c:\libs\boost\stage\lib
- ./cryptoTools/thirdparty/win/boost/stage\lib

You should check these locations can make sure that the boost library thats in the error is in fact there.

**Comment by sandy9999** [comment]
Firstly, thank you very much for your speedy response! I had changed MSVC 14.0 to MSVC 14.1 only in b2.exe (edited the original issue now, sorry for the inconvenience), and faced the same problem earlier.

I had a look at ./cryptoTools/thirdparty/win/boost/stage\lib and noticed that instead of `libboost_date_time-vc141-mt-gd-x64-1_69.lib` I had `libboost_date_time-vc141-mt-sgd-x64-1_69.lib` in that directory.

To solve this issue, I ran `.\b2.exe toolset=msvc-14.0 architecture=x86 address-model=64 --with-thread --with-filesystem --with-regex --with-date_time stage link=static variant=debug,release threading=multi` where I removed the `runtime-link=static` flag, and this created the gd files as well in that boost libs directory.

I have disabled miracl like you suggested to get rid of the other error.
